src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Trumbowyg.Edit.cshtml

```diff
@@ -53,8 +53,21 @@ else
     }
 }    
     $(function () {
-        var settings = @Html.Raw(trumbowygSettings.Options);
-        settings['lang'] = '@culture.TwoLetterISOLanguageName';
+        @if (!string.IsNullOrEmpty(trumbowygSettings.Options))
+        {
+            <text>
+            var settings = @Html.Raw(trumbowygSettings.Options);
```

Comment:
@agriffard 
Render the Options on the textarea directly in a data-options attribute like this : 

```
<textarea asp-for="Html" class="form-control" data-options="@Html.Raw(trumbowygSettings.Options);"></textarea>
```

And then here get the options value by doing : 

```
var settings = $('#@Html.IdFor(m => m.Html)').data("options");
```

Reply:
I tried doing this last time. encoding issues.

Reply:
Json.Parse(settings); ???